config/db.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import "fmt"
+
+type DbConfig struct {
```

Comment:
DB is an abbreviation. Thus, it should be either `dbConfig` or `DBConfig`. For more, please refer [this](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
Done.